components/table/test/table.vdiff.js

```diff
@@ -1001,6 +1001,33 @@ describe('table', () => {
 				await expect(elem).to.be.golden();
 			});
 
+			it('add-column', async() => {
+
+				const trs = elem.querySelectorAll('table tr');
+				for (const tr of trs) {
+					const item =  tr.querySelector('td, th');
+					const newItem = item.cloneNode();
+					newItem.innerHTML = `New ${item.innerHTML}`;
+					tr.appendChild(newItem);
+				}
+				await expect(elem).to.be.golden();
+			})
+
+			it('add-then-remove-column', async() => {
+
+				const trs = elem.querySelectorAll('table tr');
```

Comment:
To avoid repeating the same code twice, could you centralize this into a `addColumn()` helper?